modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -9,8 +9,9 @@
 
 Adding the {microshift-short} RPM package to an image builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
-* Start with step 1 to create your own minimal blueprint file which results in a faster {microshift-short} installation.
-* Start with step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
+* For a faster {microshift-short} installation, use step 1. In this step, you create your own minimal blueprint file.
+//Q: so would we use the minimal one if we want to add a lot of customization? what are the use-case differences between these?
```

Comment:
I do not know why a minimal one is necessary. I would start from what we give and change it - a single step.